backend/application/syson-application-configuration/src/main/java/org/eclipse/syson/application/services/DetailsViewService.java

```diff
@@ -443,12 +437,10 @@ public boolean setTransitionTargetParameter(TransitionUsage transitionUsage, Ele
                     .toList();
             if (succFeatMemberships.size() > 1) {
                 succFeatMemberships.get(1).getOwnedRelatedElement().stream()
-                        .findFirst()
-                        .ifPresent(feat -> feat.getOwnedRelationship().stream()
```

Comment:
Thank you for trying to improve things here, I appreciate.
Unfortunately, I'm not sure your version is better than the one existing.

Reply:
Done